enhanced/metrics.go

```diff
@@ -0,0 +1,756 @@
+			map[string]string(nil),
+		),
+	},
+	"rdsosmetrics_General_numVCPUs": {
+		Name: "numVCPUs",
+		Desc: prometheus.NewDesc(
+			"rdsosmetrics_General_numVCPUs",
+			"The number of virtual CPUs for the DB instance.",
+			[]string{"instance", "region"},
+			map[string]string(nil),
+		),
+	},
+	"rdsosmetrics_General_timestamp": {
+		Name: "timestamp",
+		Desc: prometheus.NewDesc(
+			"rdsosmetrics_General_timestamp",
+			"The time at which the metrics were taken.",
+			[]string{"instance", "region"},
+			map[string]string(nil),
+		),
+	},
+	"rdsosmetrics_General_uptime": {
+		Name: "uptime",
+		Desc: prometheus.NewDesc(
+			"rdsosmetrics_General_uptime",
+			"The amount of time that the DB instance has been active.",
+			[]string{"instance", "region"},
+			map[string]string(nil),
+		),
+	},
+	"rdsosmetrics_General_version": {
+		Name: "version",
+		Desc: prometheus.NewDesc(
+			"rdsosmetrics_General_version",
+			"The version of the OS metrics' stream JSON format.",
+			[]string{"instance", "region"},
+			map[string]string(nil),
+		),
+	},
+	"rdsosmetrics_memory_hugePagesSize": {
+		Name: "hugePagesSize",
+		Desc: prometheus.NewDesc(
+			"rdsosmetrics_memory_hugePagesSize",
+			"The size for each huge pages unit, in kilobytes.",
+			[]string{"instance", "region"},
+			map[string]string(nil),
+		),
+	},
+	"rdsosmetrics_memory_hugePagesTotal": {
+		Name: "hugePagesTotal",
+		Desc: prometheus.NewDesc(
+			"rdsosmetrics_memory_hugePagesTotal",
+			"The total number of huge pages for the system.",
+			[]string{"instance", "region"},
+			map[string]string(nil),
+		),
+	},
+	"rdsosmetrics_memory_inactive": {
+		Name: "inactive",
+		Desc: prometheus.NewDesc(
+			"rdsosmetrics_memory_inactive",
+			"The amount of least-frequently used memory pages, in kilobytes.",
+			[]string{"instance", "region"},
+			map[string]string(nil),
+		),
+	},
+	"rdsosmetrics_memory_pageTables": {
+		Name: "pageTables",
+		Desc: prometheus.NewDesc(
+			"rdsosmetrics_memory_pageTables",
+			"The amount of memory used by page tables, in kilobytes.",
+			[]string{"instance", "region"},
+			map[string]string(nil),
+		),
+	},
+	"rdsosmetrics_memory_slab": {
+		Name: "slab",
+		Desc: prometheus.NewDesc(
+			"rdsosmetrics_memory_slab",
+			"The amount of reusable kernel data structures, in kilobytes.",
+			[]string{"instance", "region"},
+			map[string]string(nil),
+		),
+	},
+	"rdsosmetrics_memory_cached": {
+		Name: "cached",
+		Desc: prometheus.NewDesc(
+			"rdsosmetrics_memory_cached",
+			"The amount of memory used for caching file system–based I/O.",
+			[]string{"instance", "region"},
+			map[string]string(nil),
+		),
+	},
+	"rdsosmetrics_memory_hugePagesFree": {
+		Name: "hugePagesFree",
+		Desc: prometheus.NewDesc(
+			"rdsosmetrics_memory_hugePagesFree",
+			"The number of free huge pages.Huge pages are a feature of the Linux kernel.",
+			[]string{"instance", "region"},
+			map[string]string(nil),
+		),
+	},
+	"rdsosmetrics_memory_hugePagesRsvd": {
```

Comment:
@AlekSi What should be name convention for those metrics? For basic metrics we have e.g. `aws_rds_swap_usage_average` should this one be called `aws_rds_memory_hugepagesrsvd` ?

Reply:
@delgod Does AWS has some advice for non-documented metrics? Are they stable? Can they be _suddenly_ removed?

Maybe we should just skip them?

Reply:
Is it the same as `node_memory_HugePages_Rsvd`? If yes, then we should use that name.

Reply:
Hmm, was not in https://confluence.percona.com/display/PMM/AWS%3A+RDS+Metrics+in+node_exporter+format

@delgod Maybe depends on env/os?

Reply:
oh, I haven't seen `node_memory_HugePages_Rsvd` metrics on CentOS, going to change our wiki